pkg/apis/alicloud/helper/helper_test.go

```diff
@@ -68,18 +67,43 @@ var _ = Describe("Helper", func() {
 	)
 
 	DescribeTable("#FindMachineImage",
-		func(configImages []api.MachineImage, name, version string, expectedMachineImage *api.MachineImage, expectErr bool) {
-			machineImage, err := FindMachineImage(configImages, name, version)
-			expectResults(machineImage, expectedMachineImage, err, expectErr)
+		func(machineImage []api.MachineImage, name, version string, encrypted bool, expectedMachineImage *api.MachineImage, expectErr bool) {
```

Comment:
```suggestion
		func(machineImage []api.MachineImage, name, version string, expectedMachineImage *api.MachineImage, encrypted, expectErr bool) {
```
```suggestion
		func(machineImage []api.MachineImage, name, version string, encrypted bool, expectedMachineImage *api.MachineImage, expectErr bool) {
```

Reply:
No, logically, expect params should be put in the behind position.